practices/risk_analysis.md

```diff
@@ -0,0 +1,82 @@
+  - task
+---
+
+# Risk Analysis
+
+**TL;DR**
+
+Risk analysis is the process of identifying and analyzing potential issues that could negatively impact key business initiatives or projects. This process is done in order to help organizations avoid or mitigate those risks.
+
+## What Is Risk Analysis
+
+Risk analysis is the process of analyzing, determining, and defining the risk of danger to government agencies, and businesses. We also use it to determine risk to individuals, towns, cities, and whole national economies.
+
+Specifically, the analysis looks at potential natural and anthropogenic events. Anthropogenic events are events that human activity causes.
+
+In IT, we commonly use a risk analysis report to align technological goals with those of the company. IT stands for information technology. Risk analysis is both quantitative or qualitative.
+
+![Process](/files/risk_analysis.jpg)
+[Wallstreetmojo.com](https://www.wallstreetmojo.com/risk-analysis/)
+
+## Why You Might Want the Risk Analysis
+1. Completion of preventative risk analysis have potential to avoid making huge losses
+2. Risk analysis will help the business avoiding security breaches
+3. New policies can be drafted to prevent potential risks from eventuating
+4. Business’ profits can also be increased by performing risk analysis.
+
+
+## How to Implement the Risk Analysis
+**1. Identify Threats**
+
+Identifying the existing and possible threats that can be faced is the very first step in Risk Analysis.And there can be multiple sources that can give rise to these threats, some of them are listed below:
+
+
+- Human – Illness, death, injury, or other loss of a key individual.
+- Operational – Disruption to supplies and operations, loss of access to essential assets, or failures in distribution.
+- Reputational – Loss of customer or employee confidence, or damage to market reputation.
+- Procedural – Failures of accountability, internal systems, or controls, or from fraud.
+- Project – Going over budget, taking too long on key tasks, or experiencing issues with product or service quality.
+- Financial – Business failure, stock market fluctuations, interest rate changes, or non-availability of funding.
+- Technical – Advances in technology, or from technical failure.
+- Natural – Weather, natural disasters, or disease.
+- Political – Changes in tax, public opinion, government policy, or foreign influence.
+
+
+Various tools can be used to uncover possible threats such as **SWOT Analysis** ,**Failure Mode** and **Effects Analysis** ,while **Scenario Analysis** helps you explore possible future threats.
+
+**2. Estimate Risk**
+
+After identification of threats that can be faced, two factors need to be calculated:
+- Likelihood of these threats being realized
+- Possible impact of these threats.
+
+So if we can estimate the exact probability of that event occuring and then multiply that by the factor that will cost us to make the things backwards in right condition.
```

Comment:
```suggestion
So if we can estimate the exact probability of that event occurring and then multiply that by the factor that will cost us to make the things backward in the right condition.
```

Things backward in the right condition is ambiguous wording. Do you mean by the Costs of the Event?